.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Astar Release Checklist
+about: Checklist to follow before creating a new release
+title: 'Release v_xx Checklist'
+labels:
+assignees: ''
+---
+
+## Required Changes (PR)
+<!---
+All the PRs that should, For Example,
+- [x] #1000
+- [x] #1006 
+-->
+
+## Release v_xx Specific
+<!---
+All the preparation specific for this release, for example
+- [x] Test batch precompile in Astar & Shiden
+- [x] Test xtokens in Astar
+-->
+
+
+## Runtime Release
+- [ ] Check Semver bumped
+     - Current: v_xx
+     - Last: v_xx
```

Comment:
This looks like a 'burden' if we have to do it manually. Shouldn't it just be a checkbox?
Like: "Shibuya semver bumped or not needed"

Reply:
@shaunxw - Make it for different crates (runtimes, collator, xcm-tools) explicitly